themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -65,56 +58,162 @@
               media="screen, print">
         <link href="${resourceUrl}/node_modules/patternfly/dist/css/patternfly-additions.min.css" rel="stylesheet"
               media="screen, print">
-        <script src="${resourceUrl}/node_modules/jquery/dist/jquery.min.js"></script>
+
         <script src="${resourceUrl}/node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
-        <script src="${resourceUrl}/node_modules/jquery-match-height/dist/jquery.matchHeight-min.js"></script>
         <script src="${resourceUrl}/node_modules/patternfly/dist/js/patternfly.min.js"></script>
-
-        <!-- Polyfill(s) for older browsers -->
-        <script src="${resourceUrl}/node_modules/core-js/client/shim.min.js"></script>
-
-        <#if properties.scripts?has_content>
-            <#list properties.scripts?split(' ') as script>
-            <script type="text/javascript" src="${resourceUrl}/${script}"></script>
-            </#list>
-        </#if>
-
-        <script src="${resourceUrl}/node_modules/zone.js/dist/zone.js"></script>
-        <script src="${resourceUrl}/node_modules/systemjs/dist/system.src.js"></script>
-
-        <script src="${resourceUrl}/systemjs.config.js"></script>
+        <script src="${authUrl}/js/keycloak.js"></script>
+
+<!--
```

Comment:
We shouldn't have comments in the templates/html. For any work that needs to be done there should be JIRAs for it, not comments.

Reply:
OK.  BTW, we do have a jira for getting rid of yarn in this code.  KEYCLOAK-6977.